composer.json

```diff
@@ -2,6 +2,7 @@
   "name": "bitbull/magento2-module-instagramwidget",
   "description": "Instagram stream for Magento2",
   "license": "proprietary",
+  "version": "1.0.1",
```

Comment:
ok understand the problem: when I moved this repository from bitbucket to github I forgot the tags. Generally we prefer to tag the repo instead of putting the tag in the composer.json of the module. Do you agree?